### ISSUE TITLE
@property shows setter only

### ISSUE THREAD

**Issue by pedrolabonia** [mkdocstrings/python#12]
**Describe the bug**
I have a class which has a `@property` getter and `@property.setter`. It doesn't show up on the new experimental parser.

**To Reproduce**
Make a class with a property setter and getter

**Expected behavior**
Expected to have a property join getter and setter and tag it as `writable`, as the old parser did, or some other form of making clear that it's a writable property

**Screenshots**
Source:

```python
class Resolve:

    @property
    def page(self) -> str:
        """Page Attribute

        Returns:
            str: Returns page name
        """
        return self._obj.GetCurrentPage()

    @page.setter
    def page(self, page: str) -> None:
        """Page Setter test

        Args:
            page (str): valid page

        Raises:
            ValueError: _description_

        Returns:
            None: None
        """        
        if page in self.pages:
            return self._obj.OpenPage(page)
        validpages = " ".join(map(str, self.pages))
        raise ValueError(
            f'"{page}" is not a valid page. Available pages are: {validpages}'
        )

```
Docs:

<img width="834" alt="CleanShot 2022-04-14 at 00 40 32@2x" src="https://user-images.githubusercontent.com/4316044/163309377-69190364-2f64-4fea-b9b5-3cd265a015b3.png">

**System (please complete the following information):**
- `mkdocstrings-python` version: [e.g. 0.6.6]
- Python version: [e.g. 3.9]
- OS: Mac OS Big Sur

**Comment by pedrolabonia** [comment]
Closing as from further investigating the code it seems it's not implemented yet in griffe as per [issue #47](https://github.com/mkdocstrings/griffe/issues/47)

**Comment by pawamoy** [comment]
Indeed, not yet implemented. Thank you for reporting and coming back to close. I'll keep improving Griffe, there're many things to do :slightly_smiling_face:
